src/components/MoviePopUp.js

```diff
@@ -0,0 +1,110 @@
+import React from "react";
+import Styled from "styled-components";
+
+const StyledInputBlock = Styled.div`
+display: flex;
+flex-direction: column;
+margin: 15px 0 0;
+`;
+
+const Styledlabel = Styled.label`
+color: #f65261;
+font-size: 18px;
+margin: 0 0 10px;
+`;
+
+const StyledInput = Styled.input`
+border-radius: 6px;
+background-color: #424242;
+border: none;
+font-size: 16px;
+padding: 10px;
+color: #fff;
+`;
+const StyledSelect = Styled.select`
+border-radius: 6px;
+background-color: #424242;
+border: none;
+font-size: 16px;
+padding: 10px;
+color: #fff;
+`;
+const StyledButtonBlock =  Styled.div`
+display: flex;
+margin: 30px 0 0;
+justify-content: flex-end;
+`
+const StyledReset = Styled.input`
+border-radius: 6px;
+background-color: transparent;
+border: 2px solid #f65261;
+font-size: 16px;
+padding: 15px 10px;
+margin: 0 15px 0 0; 
+color: #f65261;
+width: 150px;
+cursor: pointer;
+`
+const StyledSubmit = Styled.input`
+border-radius: 6px;
+background-color: #f65261;
+border: none;
+font-size: 16px;
+padding: 15px 10px;
+color:#fff;
+width: 150px;
+cursor: pointer;
+`
+
+const MoviePopUp = () => {
+
+  return (
+    <>
+   
+      <h1>addmovie</h1>
```

Comment:
usually for adding and editing item the same component is used, because most fields are the same